playbook/app/pb_kits/playbook/pb_caption/_caption.jsx

```diff
@@ -3,33 +3,33 @@
 import React from 'react'
 import classnames from 'classnames'
 import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
-import { globalProps } from '../utilities/globalProps.js'
+import { deprecatedProps, globalProps } from '../utilities/globalProps.js'
 
 type CaptionProps = {
   aria?: object,
-  className?: string,
   children: array<React.ReactNode> | React.ReactNode,
+  className?: string,
+  color?: "default" | "lighter" | "success" | "error" | "link",
```

Comment:
We are missing a  `light` color value here.

Reply:
It is a default color but I think we should still add it to match with ruby files, and it's on the list too.